Makefile

```diff
@@ -13,12 +13,12 @@ build-writeItemFunction:
 build-lambda-common:
 	npm install
 	rm -rf dist
-	echo "{\"extends\": \"./tsconfig.json\", \"include\": [\"${HANDLER}\"] }" > tsconfig-only-handler.json
+	printf "{\"extends\": \"./tsconfig.json\", \"include\": [\"${HANDLER}\"] }" > tsconfig-only-handler.json
```

Comment:
What is the reason for this change?

Reply:
WSL didn't recognize echo, problem solved with printf